targets/TARGET_STM/TARGET_STM32G4/TARGET_STM32G431xB/CMakeLists.txt

```diff
@@ -13,6 +17,7 @@ set_property(GLOBAL PROPERTY MBED_TARGET_LINKER_FILE ${CMAKE_CURRENT_SOURCE_DIR}
 
 target_sources(mbed-core
     INTERFACE
+    	system_clock.c
```

Comment:
Can you fix this, should be 2 spaces to the right as the line below

Reply:
Ah, yeah. Fixed!
(I actually had a tab there, not 4 spaces)